app/lib/dimensionsMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import { Dimensions } from 'react-native';
+
+import { dimensionsWindow, dimensionsScreen } from '../actions/dimensions';
+
+export default () => createStore => (...args) => {
+	const store = createStore(...args);
+
+	const handleDimensionsChange = (nextDimensionState) => {
+		if (nextDimensionState) {
+			if (nextDimensionState.window) {
+				store.dispatch(dimensionsWindow(nextDimensionState.window));
+			}
+			if (nextDimensionState.screen) {
```

Comment:
We may not need `screen`.
It seems to return same data as `window`.
![image](https://user-images.githubusercontent.com/804994/82832441-76468100-9e91-11ea-8b8f-15ded4fdd511.png)